src/georaster-layer-for-leaflet.ts

```diff
@@ -57,9 +57,12 @@ const GeoRasterLayer: (new (options: GeoRasterLayerOptions) => any) & typeof L.C
     updateWhenZooming: false,
     keepBuffer: 25,
     resolution: 2 ** 5,
-    debugLevel: 0
+    debugLevel: 0,
+    caching: true
   },
 
+  cache: {},
```

Comment:
I think this is actually assigning the same cache object for all instances because the prototype is shallow-copied. For each object to have its own cache instance it should be assigned in the initialize function below.

Reply:
I think you're right!